.openpublishing.publish.config.json

```diff
@@ -58,7 +58,7 @@
     {
       "path_to_root": "AspNetCore.Docs.Samples",
       "url": "https://github.com/dotnet/AspNetCore.Docs.Samples",
-      "branch": "main",
+      "branch": "serpent5-24574",
```

Comment:
I'll change this back to `main` and merge in the `AspNetCore.Docs.Samples` changes once this PR is approved.